src/render/renderer.cpp

```diff
@@ -333,8 +332,15 @@ void RendererSystem::Update( double DeltaTime )
         SetupRenderer( mWorldProjector );
         render::ParticleEngine::Get().Draw();
     }
+    if (isSuppressed)
+    {
+        glBlendFunc( GL_SRC_ALPHA, GL_ONE_MINUS_SRC_ALPHA );
+        SetupIdentity();
```

Comment:
seems redundant, SetupRenderer(mUiProjector); should overwrite everything this does